tasks/main.yml

```diff
@@ -68,19 +68,23 @@
   register: _event_delete_conf
   when: openio_event_agent_tube_delete_enabled
 
-- name: "Ensure proper state for gridinit service"
-  file:
-    name: "{{ openio_event_agent_gridinit_dir }}/{{ openio_event_agent_gridinit_file_prefix }}\
-      {{ openio_event_delete_servicename }}.conf"
-    state: "{{ 'file' if openio_event_agent_tube_delete_enabled else 'absent' }}"
-  register: _event_delete_conf
+- name: reload gridinit config
```

Comment:
```yaml
- name: "Ensure proper state for gridinit service"
  file:
    name: "{{ openio_event_agent_gridinit_dir }}/{{ openio_event_agent_gridinit_file_prefix }}\
      {{ openio_event_delete_servicename }}.conf"
    state: absent
  register: _event_delete_conf2
  when: not openio_event_agent_tube_delete_enabled

- name: restart event_agent
  shell: |
    gridinit_cmd reload
    gridinit_cmd restart  {{ openio_event_agent_namespace }}-{{ openio_event_agent_servicename }}
  when:
    - _event_agent_conf.changed or _event_agent_conf2.changed or _event_delete_conf.changed
    - not openio_event_agent_provision_only
  tags: configure

```

Is not enough ?

Reply:
Je ne sais pas.

Franchement installer un fichier pour ensuite l'effacer, je trouve ça étrange, pourquoi ne pas avoir mis la condition sur le template ?

Si tu sais comment fixer proprement ce truc vas-y